charms/istio-pilot/tox.ini

```diff
@@ -50,13 +50,13 @@ description = Apply coding style standards to code
 
 [testenv:lint]
 commands =
-    # uncomment the following line if this charm owns a lib
-    # codespell {[vars]lib_path}
+# uncomment the following line if this charm owns a lib
+# codespell {[vars]lib_path}
     codespell {toxinidir}/. --skip {toxinidir}/./.git --skip {toxinidir}/./.tox \
-      --skip {toxinidir}/./build --skip {toxinidir}/./lib --skip {toxinidir}/./venv \
-      --skip {toxinidir}/./.mypy_cache \
-      --skip {toxinidir}/./icon.svg --skip *.json.tmpl
-    # pflake8 wrapper supports config from pyproject.toml
+    --skip {toxinidir}/./build --skip {toxinidir}/./lib --skip {toxinidir}/./venv \
+    --skip {toxinidir}/./.mypy_cache \
+    --skip {toxinidir}/./icon.svg --skip *.json.tmpl
+# pflake8 wrapper supports config from pyproject.toml
```

Comment:
Do we want to remove indentation in above?
Does not seem to align with the rest of the file.